code/script.js

```diff
@@ -250,94 +262,260 @@ const setSecret = () => {
 }
 
 // This function to start (and restart) the game
+// Here we're setting charactersInPlay array to be all the characters to start with
+// What else should happen when we start the game?
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard();
+  setSecret();
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
-
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
+      value: questions.value,
       // 👆 add the value from the input here
       category: category,
     }
   } else if (category === 'eye color') {
-    // Set this up your self
+    currentQuestion = {
+      attribute: 'eyeColor',
+      value: questions.value,
+      category: category,
+      }
```

Comment:
Good code!
If you want to you could just write category, insted of categoty:category,